src/main/java/uk/ac/ebi/eva/evaseqcol/service/SeqColService.java

```diff
@@ -298,6 +295,30 @@ public List<String> getCommonFieldsDistinct(List<String> seqColAFields, List<Str
         return commonFieldsDistinct;
     }
 
+    /**
+     * Return the number of common elements between listA and listB
+     * Note: Time complexity for this method is about O(n²)*/
+    public Integer getCommonElementsCount(List<String> listA, List<String> listB) {
```

Comment:
This looks correct to me, just need to call it in the compare method!

Reply:
Done